### PR TITLE
Main PR for 1.0.0 RELEASE

### PR DESCRIPTION
**Pull request by apeltzer**
This basically merges in all changes required for TEMPLATE v1.6 stuff. Also, I created a TEMPLATE branch for future syncing of the pipeline :-) 

I also checked the output, added URLs to centralized configs and will most likely shift the `aquila` config to the centralized config repository for nf-core pipelines now. 

## PR checklist
 - [x] This comment contains a description of changes (with reason)
 - [x] If you've fixed a bug or added code that should be tested, add tests!
 - [ ] If necessary, also make a PR on the [nf-core/bacass branch on the nf-core/test-datasets repo]( https://github.com/nf-core/test-datasets/pull/new/nf-core/bacass)
 - [x] Ensure the test suite passes (`nextflow run . -profile test,docker`).
 - [x] Make sure your code lints (`nf-core lint .`).
 - [ ] Documentation in `docs` is updated
 - [ ] `CHANGELOG.md` is updated
 - [x] `README.md` is updated

### CODE REVIEW

**Comment by apeltzer** [comment]
Hi @andreas-wilm !

Can you have a look now at these changes? I guess once these are in, we could bump versions to get a first release 1.0.0 out there - I got a project in and instead of writing our own we tried using yours so this is why I'm pushing towards a first release. Basically this was just adding the newest template stuff into the pipeline, everything was already there done by you ;-) 

Thanks a bunch - lets get this rolling 🚀

File: conf/base.config [hunk 1]
```diff
@@ -1,7 +1,5 @@
 process {
 
-  container = params.container
-
   cpus = { check_max( 1, 'cpus' ) }
   memory = { check_max( 4.GB * task.attempt, 'memory' ) }
   time = { check_max( 2.h * task.attempt, 'time' ) }
```
Comment (drpatelh) [inline conf/base.config]: Replace `withName` per process to `withLabel`? Something like:
https://github.com/nf-core/rnaseq/blob/253cc1dce3c4fdf72671cb6e0540da862d4787cd/conf/base.config#L37-L48

Will have to add these labels appropriately in `main.nf` too within the relevant processes.

File: README.md [hunk 2]
```diff
@@ -1,36 +1,34 @@
...
-    * [Reference genomes](docs/configuration/reference_genomes.md)  
+    * [Local installation](https://nf-co.re/usage/local_installation)
+    * [Adding your own system config](https://nf-co.re/usage/adding_own_config)
+    * [Reference genomes](https://nf-co.re/usage/reference_genomes)
 3. [Running the pipeline](docs/usage.md)
 4. [Output and how to interpret the results](docs/output.md)
-5. [Troubleshooting](docs/troubleshooting.md)
+5. [Troubleshooting](https://nf-co.re/usage/troubleshooting)
+
+This pipeline is for bacterial assembly of next-generation sequencing reads. It can be used to quality trim your reads using [Skewer](https://github.com/relipmoc/skewer) and performs basic QC using [FastQC](https://www.bioinformatics.babraham.ac.uk/projects/fastqc/). Afterwards, the pipeline performs read assembly using [Unicycler](https://github.com/rrwick/Unicycler) and assesses assembly quality using [QUAST](http://bioinf.spbau.ru/quast). Contamination of the assembly is checked using [Kraken2](https://ccb.jhu.edu/software/kraken2/) to verify sample purity. The resulting bacterial assembly is annotated using [Prokka](https://github.com/tseemann/prokka).
```
Comment (drpatelh) [inline README.md]: Can you move this section above in the `Introduction` section or add another section called `Pipeline summary`?

Comment (apeltzer) [reply README.md]: Just done 👍

File: README.md [hunk 3]
```diff
@@ -1,36 +1,32 @@
-# nf-core/bacass
-**Simple bacterial assembly and annotation pipeline**
+# nf-core/bacass: Simple bacterial assembly and annotation pipeline
```
[the remark this answers is not in the document]
Comment (drpatelh) [reply README.md]: Logo doesnt look right!

**Pull request merged**